### ISSUE TITLE
`HSM error: object already exists` when you generate an asymmetric key

### ISSUE THREAD

**Issue by quertc** [iqlusioninc/yubihsm.rs#528]
I have the code to generate the key:

```rs
for _ in 0..number {
    match client.generate_asymmetric_key(
        0,
        label.as_str().into(),
        Domain::all(),
        Capability::SIGN_ECDSA | Capability::EXPORTABLE_UNDER_WRAP,
        EcK256,
    ) {
        Ok(_) => (),
        Err(e) => return cx.throw_error(format!("Failed to generate key: {}", e)),
    };
}
```

But when I generate, for example, 100 keys, this error appears.

Perhaps it's due to the fact that automatic generation of object_id conflicts with the already existing one. But I'm not sure. Can you help?

**Comment by tony-iqlusion** [comment]
You are hitting some limitations of the YubiHSM, regardless of what they are specifically.

It store 256 objects. Is it possible you already created that many?

**Comment by quertc** [comment]
Yes, I know about the limit, but I'm doing it on an empty Yubihsm.

**Comment by quertc** [comment]
And if this error occurs, only part of the keys are generated. But if I try again, I can generate new keys.

**Comment by tony-iqlusion** [comment]
Regardless of what you are doing, the error originates in the YubiHSM. This crate is just the messenger.

**Comment by quertc** [comment]
Yeah, I understand that. Thanks.
